index.js

```diff
@@ -48,50 +48,274 @@ const player = {
     { id: 5, name: 'Israeli', songs: [4, 5] },
   ],
   playSong(song) {
-    console.log(/* your code here */)
+    return `Playing ${song.title} from ${song.album} by ${song.artist} | ${convertSecondsToMinutes(song.duration)}.`
   },
 }
 
 function playSong(id) {
-  // your code here
+  let song = getSongById(id)
```

Comment:
This variable is redundant. You could do the entire thing in a one-liner.

Reply:
fixed, thanks!